lib/public/views/Logs/Logs.js

```diff
@@ -35,6 +35,11 @@ export default class Overview extends Observable {
 
         this.isPreviewActive = false;
         this.editors = [];
+
+        this.collapsableColumns = [];
+        this.collapsedColumns = [];
+
+        this.model.window.addEventListener('resize', () => this.notify());
```

Comment:
Wouldn't it be better to have this in the `index.html` as table resizing and stuff is on multiple screens?

Reply:
The most central place where we can put this code so it applies to all pages is the main Model itself, guess that would work as well?